src/amo/components/AddonReviewCard/styles.scss

```diff
@@ -80,6 +80,10 @@
   }
 }
 
+.flag-review-button {
```

Comment:
Because this class deals with content from the `FlagReviewMenu` component, it should be put in the scss file for that component, not in this scss file.

Reply:
Sure will move that